ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -26,6 +26,15 @@ images:
 - dockerfile_path: openshift-ci/Dockerfile.registry.build
   from: operator-registry-base
   to: performance-addon-operator-registry
+- dockerfile_path: openshift-ci/Dockerfile.registry.upstream.quay
```

Comment:
I think the Dockerfile name should be `Dockerfile.registry.upstream.dev`?

Reply:
ah, good catch, fixed